tasks/casks.yml

```diff
@@ -11,5 +11,5 @@
 
 - name: Install Homebrew Cask apps with sudo
   homebrew_cask: name={{ item }} state=present
-  sudo: yes
+  become: yes
```

Comment:
I believe `become` was first introduced in Ansible 1.9. `min_ansible_version` should be updated accordingly. (Keep in mind that other changes being introduced may require a 2.x release.)